### ISSUE TITLE
Can't transfer energy across dimensions.

### ISSUE THREAD

**Issue by Lord-Volkner** [Infinity-Evolved-Reloaded-Modpacks/Infinity-Evolved-Reloaded#171]
I've tried with Dimensional Transceivers and Tesseracts. The Tesseracts don't seem to work at all, but the Dimensional Transceivers do if I constantly go back and forth between the dimensions. When I go back to the overworld, and then return to the dimension, it will fill up with energy one time, but not refill. The Chunk loader does nothing. I haven't experimented with them in the same dimension nor I have checked anything other than energy. I never got to the point of sending items because I couldn't get power to the quarry.

The Tesseract doesn't work at all.

**Comment by SanderWxD** [comment]
Which dimensions did you try (with the Tesseract)?
The discription of your issue does make it sound like there are chunkloading issues. Which type(s) of chunkloader(s) did you use?

**Comment by Lord-Volkner** [comment]
> Which dimensions did you try (with the Tesseract)?
> The discription of your issue does make it sound like there are chunkloading issues. Which type(s) of chunkloader(s) did you use?

I tried using the Dimensional Transceiver in the mining dimension, the deep dark, and the nether, (all with the chunk loader.) It didn't work in any. I'm pretty sure that the only place I tried the Tesseract was in the deep dark (again with the chunk loader). I was pretty frustrated by that point and just cheated in a creative energy source. I hate doing that, but what else could I do? Transporting the mined items manually was a pain, but I managed.

I forget now which mod the chunk loader was from and I can't check because I tried to update the modpack in the curseforge launcher and instead of updating the modpack, it deleted my installation along with the save games. That was so disheartening that I just quit Reloaded and started playing a different modpack. Never-the-less, I hope to return to Reloaded at some point in the future and would like to see these issues fixed.

Also worth noting is that I put a chunk loader both in my base and in the dimension I was mining.

Good luck.

Thanks,
Volkner

**Comment by Lord-Volkner** [comment]
I've done some more tests to help resolve the issue. I built a wireless crafting terminal from the Wireless Crafting Terminal 2 mod and tried to access my network from The nether, the end, and the mining dimension. It doesn't work in any of them. I always get a "No Network Power" message. I built a Chunk Loader from Industrial Craft 2, placed it in my base, and selected all the necessary chunks to keep my reactor and my network fully loaded. That didn't help, still says No Network Power.

I then tested the chunk loader, which seems to have worked. I set one in the end with an RFTools Builder quarry going and left the end. When I came back, the quarry had done it's job.

I haven't tested the tesseract or DT yet in this new game, but plan to soon. I'll let you know what happens.

Volkner
